Week6

MingLin1995 opened the pull request:
林右銘
http://54.184.246.26:3000/

Commit:
Merge branch 'develop'

Commit:
Merge branch 'develop'

Commit:
Merge branch 'develop'

Commit:
feat：串接金流
 因應新需求做調整：
1.串接金流
調整項目：
1. .env.sample
2.booking.py
3.thankyou.py
4.app.py
5.TP_data.sql
6.TP_data_AWS.sql
7.booking.css
8.thankyou.css
9.booking.js
10.thankyou.js
11.booking.html
12.thankyou.html

Commit:
Merge branch 'feature/串接金流' into develop

cwpeng reviewed (changes requested):
Your thankyou page shows 查無任何訂單編號 after I checked out.

Commit:
fix：修改刪除booking邏輯
 問題：
1. 完成訂單後，不會正確顯示訂單編號
調整項目：
1.付款完成後，再刪除訂單
2.透過booking_id去查詢booking相關資訊
調整項目：
1.thankyou.py
2.booking.py

Commit:
Merge branch 'feature/串接金流' into develop

cwpeng reviewed (changes requested):
I refresh thankyou page and show 查無任何訂單編號. We should remove booking data and keep order data.

Commit:
fix：刪除booking資料，同時保存order資料
 問題：
1. 刪除booking資料時，會導致order頁面搜尋不到booking資訊
調整項目：
1. booking新增status去控制是否刪除訂單
調整項目：
1. booking.py
2.thankyou.py
3.TP_data.sql
4.TP_data_AWS.sql
5.booking.js

Commit:
Merge branch 'feature/串接金流' into develop

cwpeng reviewed (approved):
OK. Done.
